packages/ui-kit/src/components/TimeSeries/utils.ts

```diff
@@ -373,3 +374,26 @@ export function tooltipTitleCallback(context: { label: string }[], granularity:
     }[granularity] ?? title
   )
 }
+
+interface GetNumericDataOptions {
+  labels: string[]
+  values: Array<string | number>
+  log: Log
+}
+
+export function getNumericData({ labels, values, log }: GetNumericDataOptions) {
+  const newLabels = [...labels]
+  const newValues = values.map((value: string | number, index: number) => {
+    if (typeof value === 'number') return value
+
+    const displayValue = getDisplayValue({ value })
+
+    if (typeof displayValue !== 'number') {
+      log.warn('TimeSeries used with non-numeric value. Value will be filtered out:', displayValue)
```

Comment:
Let's just log once if any non-numeric value exists, and not log the specific value(s) — (there could be many of them)